automated-comments/scala/general/unnecessary_function.md

```diff
@@ -0,0 +1 @@
+Instead of using function overloading, try using an (default parameter value)[https://docs.scala-lang.org/tour/default-parameter-values.html] to set a default value for a parameter.
```

Comment:
```suggestion
Instead of using function overloading, try using a (default parameter value)[https://docs.scala-lang.org/tour/default-parameter-values.html] to set a default value for a parameter.
```